---
layout: single
title: "Project"
permalink: /project/
author_profile: false
toc: false 
show_title: false
classes: "wide"
---

Pancreatic ductal adenocarcinoma (PDAC) is an aggressive, highly metastatic and of the most therapy-resistant cancers. Poor prognosis of PDAC patients is attributed to the high heterogeneity of PDAC tumors, and to their fibrotic and immune- averse tumor microenvironment (TME). However, how the PDAC TME responds and adapts to standard therapy at the level of the 3D epigenome remains surprisingly unknown.

We propose that epigenetic changes affecting the stroma and immune compartment affect tumor cell identity (and vice versa) and explain a substantial part of PDAC therapy-resistant nature. Specifically, we postulate that changes in epigenetic marks, 3D genome organization, and the mechanical properties of the TME engage in complex crosstalk that needs to be dissected and targeted to render PDAC susceptible to therapy. To this end, we plan to a) characterize the PDAC TME at high spatial resolution, studying 3D epigenetic and mechanical properties of  cancer and stromal cells, b) screen a large selection of epigenetic drugs and modifiers, including new and repurposed compounds produced by our consortium, for their ability to reprogram tumors and improve antitumoral responsiveness of the PDAC niche, c) understand how cell-cell crosstalk is modulated and related to mechanical cues via computational simulations of multi-level  processes from chromatin reorganization to inter-cell interactions, and d) validate new combinatorial treatments in patient material.

Our endeavor requires a diverse and expansive skillset, and we brought together experts in 3D-(epi)genomics, integrative bioinformatics, mechanobiology, computational modeling, drug screening and PDAC biology from six countries and three continents. Together, we will study patient material by spatially-resolved proteomics, bulk transcriptomics and computational network-based integration to look at the distribution and levels of cell identity and epigenetic markers, and PDAC TME topology. We will also exploit our clinically-annotated collection of patient-derived organoids in co-cultures with autologous fibroblasts to screen compounds targeting the epigenome, signaling and mechanobiology pathways and assess the response of each TME compartment. For those compounds showing most promise, we will perform extensive characterization via epigenomic and 3D chromatin interaction profiling in organoids. The outcome of the screen and the epigenomic characterization will then feed into a novel simulation framework combining epigenetic and mechanical properties of TME cells and matrix to model inter-cell communication to predict responses. Last, selected epidrugs in combination with standard chemotherapy will be validated in ex vivo-grown precision-cut tumor slices.

We expect our unique approach to define new regimes for modulating the PDAC TME in favor of therapy and to identify epigenomic and mechanosensitive dependencies, allowing for precise patient stratification and paving the way for new therapies.